db-pull-on-boot/templates/db-pull.sh.j2

```diff
@@ -1,4 +1,24 @@
 #!/bin/bash
+LOCK_FILE="/tmp/blockchain-backup.lock"
+
+cleanup() {
+	rm -f "$LOCK_FILE"
+	rm -f /tmp/*.sqlite
```

Comment:
Not sure if we need this line - this I think is only relevant to the database backup?
```suggestion
```